### PR TITLE
docs: add documentation for Cube Store

### PR DESCRIPTION
**Pull request by ovr**
Hello!

Thanks

### CODE REVIEW

File: docs/content/Caching/Using-Pre-Aggregations.md [hunk 1]
```diff
@@ -147,3 +147,117 @@ CUBEJS_EXT_DB_USER=<YOUR_DB_USER_HERE>
...
+
+<!-- prettier-ignore-start -->
+[[info | ]]
+| You should use CUBEJS_DEV_MODE=true and EXTERNAL_DB variables should not be defined.
+<!-- prettier-ignore-end -->
+
+Starting from `v0.26.48` version, Cube.js ships with automatically provisioning for Cube Store in `CUBEJS_DEV_MODE`. You don't need to set up
+any `EXTERNAL_DB` variables or `externalDriverFactory` inside your `cube.js` configuration file.
+
+For versions before `v0.26.48`, You should upgrade your project to latest version and install a driver for Cube Store:
```
Comment (rpaik) [inline docs/content/Caching/Using-Pre-Aggregations.md:204]:
```suggestion
For versions before `v0.26.48`, You should upgrade your project to the latest version and install a driver for Cube Store:
```

File: docs/content/Caching/Using-Pre-Aggregations.md [hunk 2]
```diff
@@ -147,3 +147,117 @@ CUBEJS_EXT_DB_USER=<YOUR_DB_USER_HERE>
...
+| `x86_64-linux-gnu`                   |   ✓    |
+| `x86_64-linux-musl`                  |   ✓    |
+| `x86-linux-gnu`                      |  N/A   |
+| `x86-linux-musl`                     |  N/A   |
+| `x86_64-darwin`                      |   ✓    |
+| `arm64-darwin` [1]                   |   ✓    |
+| `x86_64-win32`                       |   ✓    |
+| `x86-win32`                          |  N/A   |
+
+[1] It can be launched by Rosseta 2 via `x86_64-apple` binary.
```
Comment (rpaik) [inline docs/content/Caching/Using-Pre-Aggregations.md:190]:
```suggestion
[1] It can be launched by Rosseta 2 via the `x86_64-apple` binary.
```

**Review by rpaik — CHANGES_REQUESTED** [review]
Just a few minor edits 😄

**Review by rpaik — APPROVED** [review]

**Comment by hassankhan** [comment]
Closing PR in favor of https://github.com/cube-js/cube.js/pull/2463, thanks for doing the groundwork on this @ovr 👍

**Pull request closed without merging**